edq/cli/config/set.py

```diff
@@ -0,0 +1,81 @@
+"""
+Set a configuration option.
+"""
+
+import argparse
+import sys
+import typing
+
+import edq.core.argparser
+import edq.core.config
+
+def run_cli(args: argparse.Namespace) -> int:
+    """ Run the CLI. """
+
+    config: typing.Dict[str, str] = {}
```

Comment:
Name needs more context.
`config` is a bad name because it does not contain current configuration options.
`config_to_set` (as you have in the args) is fine.